### DISCUSSION TITLE
Code Server Online with C/C++ extension(ver1.1.1) cause CPU 100%usage

### DISCUSSION THREAD

**Discussion by KannaNyaa** [coder/code-server#2296] [Q&A] [answered]
After the C/C++ (ver1.1.1) came out, My Code Server updated it from 1.1.0 to 1.1.1 in default.
After a long time, it had succeeded.(My Online VM has 1 core, 2G Ram, 5Mb download speed)
However the Code Server's terminal started showing:
 `/home/ubuntu/.local/share/code-server/extensions/ms-vscode.cpptools-1.1.1/bin/cpptools: /home/ubuntu/.local/share/code-server/extensions/ms-vscode.cpptools-1.1.1/bin/cpptools: cannot execute binary file.`
Is that meaning I should run My Code Server in `su` permission?
And Code Server IDE shows:
 `The C/C++ extension failed to install successfully. You will need to repair or reinstall the extension for C/C++ language features to function properly.`
My VM is stuck by the Code Server with 100% CPU usage...
It has never happened!(before the extension updated) .
I reboot the VM and reload the extension. It was USELESS! The Warnings come again!
What can I do for solving this problem? :(
Kid can't live without programming.
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;T_T 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(T＿T)
&nbsp;&nbsp;&nbsp;&nbsp;o(╥﹏╥)o 
~~o(>_<)o ~~

**Comment by code-asher** [comment, marked as answer]
You might be running into https://github.com/cdr/code-server/issues/2120.

If so, for now you'll have to manually download the vsix for your
architecture from the vscode-cpptools GitHub repository and upload it to
code-server.

**Reply by KannaNyaa** [reply]
Thx, I'll have a try. I just disabled the update function and it can be used temporarily.
